<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="6" hostname="DESKTOP-KJC6B92" failures="1" tests="2" name="Class3And2(failed)(failed)(failed)" time="0.077" errors="0" timestamp="2024-09-04T07:25:01 IST">
  <testcase classname="pack1.TestClass3" name="TestCase100" time="0.007"/>
  <testcase classname="pack1.TestClass2" name="TestCase5" time="0.006">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at pack1.TestClass2.TestCase5(TestClass2.java:32)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- TestCase5 -->
  <testcase classname="pack1.TestClass2" name="TestCase4" time="0.0">
    <ignored/>
  </testcase> <!-- TestCase4 -->
  <testcase classname="pack1.TestClass3" name="TestCase9" time="0.0">
    <ignored/>
  </testcase> <!-- TestCase9 -->
  <testcase classname="pack1.TestClass3" name="TestCase200" time="0.0">
    <ignored/>
  </testcase> <!-- TestCase200 -->
  <testcase classname="pack1.TestClass3" name="TestCase8" time="0.0">
    <ignored/>
  </testcase> <!-- TestCase8 -->
  <testcase classname="pack1.TestClass3" name="TestCase7" time="0.0">
    <ignored/>
  </testcase> <!-- TestCase7 -->
  <testcase classname="pack1.TestClass2" name="TestCase6" time="0.0">
    <ignored/>
  </testcase> <!-- TestCase6 -->
</testsuite> <!-- Class3And2(failed)(failed)(failed) -->
